tests/SendTestMailTest.php

```diff
@@ -79,4 +80,41 @@ public function it_will_throw_an_exception_when_passing_invalid_values()
             '--values' => 'myInteger',
         ]);
     }
+
+    /** @test */
+    public function it_will_use_base_namespace_variable_if_class_is_not_found()
+    {
+        Config::set('mailable-test.base_namespace', 'Spatie\\MailableTest\\Test');
```

Comment:
Preferably users shouldn't have to type double slashes in the config file. Could change this line to

```
config('mailable-test.base_namespace', Spatie\MailableTest\Test::class);
```

and make sure the test passes.